recipe/meta.yaml

```diff
@@ -10,27 +10,26 @@ source:
     sha256: 156eab31d9f9bac218bbd391559c667848372a5c584472784695e4fac087fd5b
 
 build:
-  number: 0
-  skip: true  # [win]
-  skip: true  # [py<37]
+  number: 1
   skip: true  # [win and python_impl=="pypy"]
 
 requirements:
   build:
     - python                              # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform and not (build_platform == "linux-64" and target_platform == "win-64")]
```

Comment:
Can you add a comment why this and `compiler('c')` is excluded in the win-cross situation?